addons/binding/org.openhab.binding.avmfritz/src/main/java/org/openhab/binding/avmfritz/handler/DeviceHandler.java

```diff
@@ -0,0 +1,244 @@
+ * 
+ */
+public class DeviceHandler extends BaseThingHandler 
+implements IFritzHandler {
+	/**
+	 * Logger
+	 */
+	private final Logger logger = LoggerFactory.getLogger(this.getClass());
+
+	/**
+	 * Ip of PL546E in standalone mode
+	 */
+	private String soloIp;
+	/**
+	 * the refresh interval which is used to poll values from the fritzaha.
+	 * server (optional, defaults to 15 s)
+	 */
+	protected long refreshInterval = 15;
+	/**
+	 * Interface object for querying the FRITZ!Box web interface
+	 */
+	protected FritzahaWebInterface connection;
+	/**
+	 * Job which will do the FRITZ!Box polling
+	 */
+	private DeviceListPolling pollingRunnable;
+	/**
+	 * Schedule for polling
+	 */
+	private ScheduledFuture<?> pollingJob;
+	
+	public DeviceHandler(Thing thing) {
+		super(thing);
+		this.pollingRunnable = new DeviceListPolling(this);
+	}
+	/**
+	 * Initializes the bridge.
+	 */
+	@Override
+	public void initialize() {
+		if( this.getThing().getThingTypeUID().equals(PL546E_STANDALONE_THING_TYPE) ) {
+			logger.debug("About to initialize thing "
+					+ BindingConstants.DEVICE_PL546E_STANDALONE);
+			Thing thing = this.getThing();
+			AvmFritzConfiguration config = this.getConfigAs(AvmFritzConfiguration.class);
+			this.soloIp = config.getIpAddress();
+			
+			logger.debug("discovered PL546E initialized: "
+					+ config.toString());
+
+			this.refreshInterval = config.getPollingInterval();
+			this.connection = new FritzahaWebInterface(config, this);
+			if (config.getPassword() != null) {
+				this.onUpdate();
+			} else {
+				thing.setStatusInfo(new ThingStatusInfo(
+						ThingStatus.OFFLINE, 
+						ThingStatusDetail.CONFIGURATION_ERROR, 
+						"no password set"));
+			}
+		}
+	}
+
+	/**
+	 * Disposes the thing.
+	 */
+	@Override
+	public void dispose() {
+		if( this.getThing().getThingTypeUID().equals(PL546E_STANDALONE_THING_TYPE) ) {
+			logger.debug("Handler disposed.");
+			if (pollingJob != null && !pollingJob.isCancelled()) {
+				pollingJob.cancel(true);
+				pollingJob = null;
+			}
+		}
+	}
+
+	/**
+	 * Start the polling.
+	 */
+	private synchronized void onUpdate() {
+		if (this.getThing() != null) {
+			if (pollingJob == null || pollingJob.isCancelled()) {
+				logger.debug("start polling job at intervall " + refreshInterval);
+				pollingJob = scheduler.scheduleWithFixedDelay(pollingRunnable, 1,
+						refreshInterval, TimeUnit.SECONDS);
+			} else {
+				logger.debug("pollingJob active");
+			}
+		} else {
+			logger.warn("thing is null");
+		}
+	}
+	/**
+	 * Handle the commands for switchable outlets.
+	 */
+	@Override
+	public void handleCommand(ChannelUID channelUID, Command command) {
+		logger.debug("command for " + channelUID.getAsString() + ": "
+				+ command.toString());
+		if (channelUID.getId().equals(CHANNEL_SWITCH)) {
+			logger.debug("update " + channelUID.getAsString() + " with "
+					+ command.toString());
+			Bridge bridge = this.getBridge();
+			if (bridge != null
```

Comment:
For the 546E standalone, this never gets executed. Am I missing something or is this a bug?